.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - ./gradlew assemble testClasses --stacktrace
 script:
   # only build and deploy the results if the build is performed on the master branch
-#  - if scripts/travis-tex-run-required.sh ; then scripts/generate-and-deploy-report.sh; fi
+  - if scripts/travis-tex-run-required.sh ; then scripts/generate-and-deploy-report.sh; fi
```

Comment:
Does this report work? It's not tested by this PR, right?

Reply:
The point if this is to deploy the homepage to http://docs.inf.mit.bme.hu/ingraph/. The tech report will not work anytime soon, so I'll only invoke deploy.sh.